packages/compass-indexes/src/modules/create-index/index.js

```diff
@@ -165,14 +160,13 @@ export const createIndex = () => {
     });
 
     const options = {};
-    options.background = state.isBackground;
     options.unique = state.isUnique;
     // The server will generate a name when we don't provide one.
     if (state.name !== '') {
       options.name = state.name;
     }
     if (state.isCustomCollation) {
-      options.collation = state.collation;
+      options.collation = state.collationInfo.value;
```

Comment:
As I mentioned in https://github.com/mongodb-js/compass/pull/3298#discussion_r932990110 you probably don't need to keep the value in the state and always parse / validate the value on edit, this is the only place where value is used and so it would make sense to do the parsing / validation part here. You probably might want to add validation action dispatch similar to all the other fields in the form here too, so something like this:

```suggestion
      options.collation = isCollationValid(collationString);
      if (!options.collation) {
        dispatch(handleError('Collation is not valid'))
        return;
      }
```